app/uk/gov/hmrc/tradergoodsprofilesdatastore/models/CheckRecords.scala

```diff
@@ -18,21 +18,29 @@ package uk.gov.hmrc.tradergoodsprofilesdatastore.models
 
 import play.api.libs.json.{Format, JsSuccess, JsValue, Json, Reads, Writes}
 
+import java.time.Instant
+
 case class CheckRecords(
-  eori: String
+  eori: String,
+  recordsUpdating: Boolean,
+  lastUpdated: Instant
 )
 
 object CheckRecords {
 
   implicit val writes: Writes[CheckRecords] = (checkRecords: CheckRecords) =>
```

Comment:
Since you're just using the names of fields in the class you can replace your reads and writes with the format macro:

```scala
lazy val format: OFormat[CheckRecords] = Json.format
```